aries_cloudagent/admin/server.py

```diff
@@ -523,6 +524,27 @@ async def plugins_handler(self, request: web.BaseRequest):
         plugins = registry and sorted(registry.plugin_names) or []
         return web.json_response({"result": plugins})
 
+    @docs(tags=["server"], summary="Fetch the server configuration")
+    @response_schema(AdminStatusSchema(), 200, description="")
+    async def config_handler(self, request: web.BaseRequest):
+        """
+        Request handler for the server configuration.
+
+        Args:
+            request: aiohttp request object
+
+        Returns:
+            The web response
+
+        """
+        return web.json_response(
+            {
+                k: self.context.settings[k]
+                for k in self.context.settings
+                if k != "admin.admin_api_key"
```

Comment:
Other parameters that should be filtered:  wallet key, wallet auth attributes (if using postgres), web hooks (if they contain api keys), wallet seed

Reply:
great, I'll amend accordingly

Reply:
Are wallet auth attributes `wallet.storage.creds` ? Are there others?